app/models/ecf_participant_validation_data.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class ECFParticipantValidationData < ApplicationRecord
+  has_paper_trail
```

Comment:
Don't think we should have this one as it's PII and in theory we want to get rid if this as soon as we identify the person